libs/EXTERNAL/libargparse/src/argparse.cpp

```diff
@@ -4,6 +4,7 @@
 #include <cassert>
 #include <string>
 #include <set>
+#include <limits>
```

Comment:
This looks like a leftover

Reply:
But it's not. It's actually required as without it compilation using Clang on the recent GH ubuntu-latest VM fails.

It's the same issue that was hitting `symbiflow-yosys-plugins` but here we can solve it properly by adding the missing include to the source code (for plugins we couldn't as we just clone and build Yosys there).

Reply:
Aha, right, I didn't notice the commit's comment. (good that it was only a one line change to fix the Clang compilation issue)